bin/export_co.py

```diff
@@ -110,7 +111,10 @@
         co = hh[1]['CO'][:]
         hh.close()
     else:
-        co = sp.eye(dd.size)
+        w = rr>0.
+        co = sp.zeros_like(dd)
+        co[w] = dd[w]*coefDD/(coefDD*rr[w])**2
+        co = sp.diag(co)
 
     ### Save
     h = fitsio.FITS(args.out,'rw',clobber=True)
```

Comment:
I know this is not part of this PR, but it might be useful to keep DD, DR and RR in the output file. Note that the fitter won't be bothered by the extra columns or extra HDUs...